### DISCUSSION TITLE
BACnet Version for bacpypes version of 0.18.1

### DISCUSSION THREAD

**Discussion by JimitD** [JoelBender/bacpypes#383] [Q&A]
Hi Joel,

can you please let me know the BACnet version of bacpypes version 0.18.1? Can we check this in system?

**Comment by JoelBender** [comment]
The current version is **0.18.3** that is in the _master_ branch and on [PyPI](https://pypi.org/project/bacpypes/).  If you are looking for the downloads that are on the [releases](https://github.com/JoelBender/bacpypes/releases) list then it looks like I haven't been keeping up on the maintenance of that project component.

**Reply by JimitD** [reply]
Thanks, Joel,

But that tells me version of bacpypes. But, I need version of BACnet on which it is developed.

**Reply by JimitD** [reply]
Probably from below  website, which one we are using? 

http://www.bacnet.org/Addenda/

Version Number | = 1
Revision Number | = 22

135-2016 (BACnet-2016, Ver. 1, Rev. 19)
135-2012 (BACnet-2012, Ver. 1, Rev. 14)
135-2010 (BACnet-2010, Ver. 1, Rev. 12)

**Reply by JimitD** [reply]
HI Joel,

Your help is appreciated. 

Thanks

**Reply by ChristianTremblay** [reply]
I think the answer is "somewhat between rev 14 and rev 19".
